app/presenters/project_show_presenter.rb

```diff
@@ -148,7 +148,8 @@ def quota_percentage(session_id:)
 
   def user_has_access?(user:)
     return true if user.eligible_sysadmin?
-    data_sponsor.uid == user.uid || data_manager.uid == user.uid || data_users.include?(user.uid)
+    return true if data_sponsor.uid == user.uid || data_manager.uid == user.uid
+    data_users.find { |data_user| data_user.uid == user.uid } != nil
```

Comment:
I think the include? easier to read.  Could you change the line to:
```suggestion
     data_users.map(&:uid).include?(user.uid)
```

Or I wonder if data_users.include?(user) would work?